lms/envs/aws.py

```diff
@@ -191,6 +191,13 @@
 CONTACT_EMAIL = ENV_TOKENS.get('CONTACT_EMAIL', CONTACT_EMAIL)
 BUGS_EMAIL = ENV_TOKENS.get('BUGS_EMAIL', BUGS_EMAIL)
 PAYMENT_SUPPORT_EMAIL = ENV_TOKENS.get('PAYMENT_SUPPORT_EMAIL', PAYMENT_SUPPORT_EMAIL)
+PAYMENT_CONFIRM_EMAIL = ENV_TOKENS.get(
+    'PAYMENT_CONFIRM_EMAIL',
+    ENV_TOKENS.get(
+        'PAYMENT_SUPPORT_EMAIL',
+        PAYMENT_CONFIRM_EMAIL
```

Comment:
Why the crazy double level .get here? Are you trying to coalesce the two values?